src/System.Private.CoreLib/shared/System/Reflection/InterfaceMapping.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Hmm.  I'm not sure what to do with this one.  Technically all the members could be null.  But it looks like our implementations that create these always initialize all four fields to be non-null.  Could you tag this with tag this with a `// TODO-NULLABLE: Re-review` for us to follow-up on it again later?

Reply:
Sure

Reply:
@stephentoub IMO all reference types in structs should be nullable because they must have parameterless constructor which doesn't do anything

Reply:
@krwq as I added `// TODO-NULLABLE: Re-review` comment you can say your opinion when we re review, so merging the PR